### PR TITLE
Optimise hero image

### PR DESCRIPTION
**Pull request by mattkelly4**
## Summary
Reduced size of hero image to improve loading time of Home page.

## Changes
**Fixes**:
- Optimised hero image

### CODE REVIEW

**Comment by mattkelly4** [comment]
Please delete branch after merging.

**Review by chrisg06 — APPROVED** [review]

**Review by alphadelta332 — APPROVED** [review]

**Review by tylerthetiletiler — APPROVED** [review]

**Comment by tylerthetiletiler** [comment]
@mattkelly4 I'll let you merge and do what you need to do

**Pull request merged**
